### PR TITLE
FEC-11578 AdLayout config for LiveMedia

### PR DESCRIPTION
**Pull request by GouravSna**
This PR is fixing FEC-11938 as well.

### CODE REVIEW

File: playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt [hunk 1]
```diff
@@ -484,6 +562,39 @@ class PKAdvertisingController: PKAdvertising, IMAEventsListener {
...
 
+    /**
+     * Countdown timer only for Live medias
+     * This timer helps to calculate the time left for the next midroll
+     * Timer has a flag which becomes true when the timer finishes
+     * This flag is being used in `playHeadUpdated` event to trigger the midroll
+     */
+    private fun createTimerForLiveMedias() {
+        log.d("createTimerForLiveMedias")
+        liveMediaCountdownTimer = object: ResumableCountDownTimer(midrollFrequency, 100) {
```
Comment (GouravSna) [inline playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt]: Make 100 as constant

File: playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt [hunk 2]
```diff
@@ -363,35 +422,54 @@ class PKAdvertisingController: PKAdvertising, IMAEventsListener {
...
             log.d("Player seeking for player position = ${player?.currentPosition} - currentPosition ${it.currentPosition} - targetPosition ${it.targetPosition}" )
             isPlayerSeeking = true
             if (isLiveMedia()) {
-                log.v("Seeking Event. For Live Medias only Preroll ad will be played. Hence dropping other Ads if configured.")
                 return@addListener
             }
         }
 
         messageBus?.addListener(this, PlayerEvent.loadedMetadata) {
-            log.d("loadedMetadata Player duration is = ${player?.duration}" )
```
Comment (GouravSna) [inline playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt:382]: Recheck it.

Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt:382]: Done moved the code to check `isLive` inside `checkTypeOfMidrollAdPresent` update method

File: playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt (hunk 1 above)
Comment (GouravSna) [reply playkit/src/main/java/com/kaltura/playkit/ads/PKAdvertisingController.kt]: Done

**Review by giladna — APPROVED** [review]

**Pull request merged**